test_logwarn

```diff
@@ -97,6 +97,7 @@ is-ignored '\[warn\].* Unable to wakeup scheduler: Request timeout'
 is-ignored '\[warn\].* fatal: Invalid revision range .*\.\.'
 is-ignored '\[warn\] \[pid:[0-9]*\] $'
 is-ignored '\[error\].*cmd returned 32768$'
+is-ignored '\[error\] Worker 17519 has no heartbeat \(900 seconds\), restarting \(see FAQ for more\)'
```

Comment:
Changing that to
```
is-ignored '\[error\] Worker 17519 has no heartbeat'
```
fails the test.
I suggest to do a very minimal check in the test itself. No need to include the rest of the line. Looking for `heartbeat` would actually be enough. The `\(` is probably doing the wrong thing.